core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -138,6 +142,45 @@ oppia.factory('SimpleEditorManagerService', [
         data.questionList.addQuestion(QuestionObjectFactory.create(
           lastStateName, stateData.interaction, ''));
       },
+      changeQuestion: function(type, index) {
+        var currentStateName = (
+          index === 0 ? 'Introduction' : 'Question ' + index);
+        var questionCount = data.questionList._questions.length;
```

Comment:
Instead of referring to private attributes, can we call methods like data.questionList.getNumQuestions()? If those methods don't exist, feel free to add them. Ditto below.